### ISSUE TITLE
Ubuntu 18.04 

### ISSUE THREAD

**Issue by sandra1n** [olav-st/screencloud#318]
I like your app.
But in new Ubuntu

```
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04 LTS
Release:	18.04
Codename:	bionic
```

I have Dual Monitors (1 vertical)

There is some problems (when i do screenshot selection or full sreenshot). That is screen of my browser with github creating new issue....as you see,  the picture moved left and up :

![image](https://user-images.githubusercontent.com/1708304/42930076-b214eeea-8b44-11e8-80da-acf753fe99b3.png)

With this monitor config i haven't issues when i'm used 16.04, 16.10 or 14.04

How i can fix it?

_And small question ...i had issue when i can't added app in autostart (in settings was enabled this feature, but not working). How to fix this?_

Thanks.

**Comment by synweap15** [comment]
Love ScreenCloud, but on 18.04 the issue is similar - but in my case the second screen goes all black, and the primary screen works okay.

**Comment by fliespl** [comment]
Seems like it's resolved in 1.4.0

**Comment by synweap15** [comment]
@fliespl confirmed to be fixed in 1.4.0
